.travis.yml

```diff
@@ -1,48 +1,33 @@
-# Run `travis lint` when changing this file to avoid breaking the build.
-
 cache:
   directories:
-    - $HOME/.m2
-
+  - "$HOME/.m2"
 language: java
-
 jdk: openjdk11
-
 before_install:
-  # allocate commits to CI, not the owner of the deploy key
-  - git config user.name "zipkinci"
-  - git config user.email "zipkinci+zipkin-dev@googlegroups.com"
-  # setup https authentication credentials, used by ./mvnw release:prepare
-  - git config credential.helper "store --file=.git/credentials"
-  - echo "https://$GH_TOKEN:@github.com" > .git/credentials
-
-# Override default travis to use the maven wrapper; skip license on travis due to #1512
-install: ./mvnw install -DskipTests=true -Dlicense.skip=true -Dmaven.javadoc.skip=true -B -V
-script: ./travis/publish.sh
-
-# Don't build release tags. This avoids publish conflicts because the version commit exists both on master and the release tag.
-# See https://github.com/travis-ci/travis-ci/issues/1532
+- openssl aes-256-cbc -K $encrypted_80d2740393f3_key -iv $encrypted_80d2740393f3_iv
+  -in travis/zipkin-gcp-ci-0d7917f58da7.json.enc -out travis/zipkin-gcp-ci-0d7917f58da7.json -d
+- export GOOGLE_APPLICATION_CREDENTIALS=travis/zipkin-gcp-ci-0d7917f58da7.json
+- git config user.name "zipkinci"
+- git config user.email "zipkinci+zipkin-dev@googlegroups.com"
+- git config credential.helper "store --file=.git/credentials"
+- echo "https://$GH_TOKEN:@github.com" > .git/credentials
+install: "./mvnw install -DskipTests=true -Dlicense.skip=true -Dmaven.javadoc.skip=true
+  -B -V"
+script: "./travis/publish.sh"
 branches:
   except:
-    - /^[0-9]/
-
+  - "/^[0-9]/"
 notifications:
   webhooks:
     urls:
-      - https://webhooks.gitter.im/e/ead3c37d57527214e9f2
-      - https://webhooks.gitter.im/e/e57478303f87ecd7bffc
+    - https://webhooks.gitter.im/e/ead3c37d57527214e9f2
+    - https://webhooks.gitter.im/e/e57478303f87ecd7bffc
     on_success: change
     on_failure: always
-
 env:
   global:
-    # Ex. travis encrypt BINTRAY_USER=your_github_account
```

Comment:
Looks like we lost these comments which are usefulI I think. The new secret could have a comment about what the name of the service account is.